src/vl-tabs.js

```diff
@@ -100,7 +104,7 @@ export class VlTabs extends vlElement(HTMLElement) {
     const pathname = window.location.pathname;
     return this._template(`
       <li is="vl-tab" data-vl-href="${pathname}#${id}" data-vl-id="${id}">
-        ${(title)}
+        <slot name="${id}-title-slot">${(title)}</slot>
```

Comment:
Past dit slot niet beter in de vl-tab-pane component? Dan moet je ook niet met `${id}` prefixen werken in de slot name.
De suffix kan in principe ook weg, aangezien je normaal wel weet dat het over een slot gaat als gebruiker.

Reply:
Dit slot komt opgevuld uit de vl-tab-pane indien een slotted tag is voorzien anders wordt de titel uit het attribuut genomen en in deze slot gestopt. 
De vl-tab die je hier ziet zet dan weer wat de vl-tab-pane aanbied om in een titel in de vorm van een link.